modules/oe_translation_cdt/src/TranslationRequestCdtInterface.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+namespace Drupal\oe_translation_cdt;
+
+use Drupal\oe_translation_remote\TranslationRequestRemoteInterface;
+
+/**
+ * An interface for CDT bundle class for oe_translation_request entities.
+ */
+interface TranslationRequestCdtInterface extends TranslationRequestRemoteInterface {
+
+  /**
+   * Returns the CDT ID.
+   *
+   * @return string|null
+   *   The CDT ID including request year, or NULL if not set.
```

Comment:
lets mention the format or an example rather than "including request year , or NULL if not set."